pandas/tests/apply/test_frame_apply.py

```diff
@@ -131,6 +131,13 @@ def test_apply_funcs_over_empty(self, func):
         expected = getattr(df, func)()
         tm.assert_series_equal(result, expected)
 
+    def test_apply_return_type_none(self):
+        df = DataFrame([["orig1", "orig2"], ["orig3", "orig4"]])
```

Comment:
can you add the issue number as a comment

Reply:
Yes, sure.